ppl/zqd/ingest/multipartlogreader.go

```diff
@@ -57,14 +70,14 @@ read:
 			if m.stopErr {
 				return nil, err
 			}
-			m.appendWarning(zr, err)
+			m.appendWarning(zr.(*zbuf.File), err)
 		}
 		goto read
 	}
 	return rec, err
 }
 
-func (m *MultipartLogReader) next() (*zbuf.File, error) {
+func (m *MultipartLogReader) next() (zbuf.ReadCloser, error) {
```

Comment:
Can you explain why you needed to change this return type?

Reply:
Because this might be passed to driver.NewReader which expects a zbuf.ReadCloser.